### DISCUSSION TITLE
Is it possible to control the quality of youtube video player?

### DISCUSSION THREAD

**Discussion by mikeyhodl** [vidstack/player#1344] [Q&A] [answered]
I have a nextjs application that plays youtube videos, is there a way to add a media player component to change video playback quality options 
![image](https://github.com/vidstack/player/assets/49108305/00fc3e1c-2660-4068-8680-fa9952a694bc)

**Comment by mihar-22** [comment, marked as answer]
Nope, there's no API for it on the YouTube embed.

**Reply by mikeyhodl** [reply]
okay
